### DISCUSSION TITLE
Font Size becomes smaller in version 6.4.3 in UMINHO template

### DISCUSSION THREAD

**Discussion by vascoalramos** [joaomlourenco/novathesis#183] [Q&A] [answered]
Hi Professor.

I noticed that the font size became smaller, comparing to the previous version I was using.
![image](https://user-images.githubusercontent.com/38591616/119378686-6f299d00-bcb6-11eb-9825-586190404fed.png)
I was able to "fix" it by adding `\normalsize` at the end. But wanted to ask if that change of font size is on purpose or not and thought it could be useful.

Thanks

**Comment by b-pereira** [comment]
It shouldn't work that way. The `\small` macro was to be used only with the URL. 

What it should be:
```tex 
\normalsize{ stuff }
\small{\url{doclicenseURL}}
```
Because the \small tag isn't enclosed in curly braces it will apply to all the document from that point on.

**Put this as an issue using the bug template**: You can and you should fix it and request a pull request. Or I can do that for you.

**Reply by vascoalramos** [reply]
Thanks, didn't knew that, I'll do it then!

**Reply by b-pereira** [reply]
OK. The issue is created. Now to add changes, I don't think how could you have permissions to change the code. @joaomlourenco has the say on that.

**Reply by vascoalramos** [reply]
I tried the approach you said, with curly brackets, but it doesn't work. The `\small` continues to be applied to the whole document from that point forward.

**Comment by joaomlourenco** [comment, marked as answer]
What I believe it should be, but I can't test now:
```latex
\newcommand{\licenseThis}{%
  \normalsize%
  \textit{\textbf{\theumcopyrightlabel(\option{/novathesis/coverlang})}}\\
  \doclicenseImage\\[0.5mm]%
  \textbf{\doclicenseLongName}\\[0.5mm]
  \textbf{\doclicenseName}\\[2ex]
  {\small\url{\doclicenseURL}}
}
```

**Reply by vascoalramos** [reply]
Yes, I was just trying that and indeed it worked, thank you!

**Reply by vascoalramos** [reply]
I'll create a PR then, thank you

**Reply by joaomlourenco** [reply]
Fixed.
